visualization/app/codeCharta/ui/searchPanel/mapTreeView/mapTreeViewLevel/mapTreeViewLevel.component.spec.ts

```diff
@@ -126,7 +126,7 @@ describe("mapTreeViewLevel", () => {
 		detectChanges()
 
 		const bigLeaf = container.querySelector("#\\/root\\/bigLeaf")
-		const isAngularGreen = bigLeaf.querySelector(".angular-green")
+		const isAngularGreen = bigLeaf.querySelector(".tree-search-result")
```

Comment:
we dont search for green anymore so the name of the const should be adjusted aswell